dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -34,10 +34,10 @@
     %p Edit Markdown:
 
     %textarea#level_markdown_textarea
-    #markdown-preview
+    #level_markdown_textarea_preview
 
   .markdown.teacher
     %p Edit Teacher Only Markdown:
 
     %textarea#level_teacher_markdown_textarea
-    #teacher-markdown-preview
+    #level_teacher_markdown_textarea_preview
```

Comment:
Makes sense.  Not a big deal, just thought I'd call it out.  I don't know what the reasoning behind this is; [Google's styleguide](https://google.github.io/styleguide/htmlcssguide.xml?showone=ID_and_Class_Name_Delimiters#ID_and_Class_Name_Delimiters) that we've tried to adopt just says "in order to improve understanding and scannability."  Maybe just to disambiguate from variables (which can't contain hyphens)?